LSPS2/README.md

```diff
@@ -0,0 +1,864 @@
+    the rest of the parts verbatim, if any.
+  * Else subtract `opening_fee = opening_fee - (part_amount - htlc_minimum_msat)`,
+    then forward `htlc_minimum_msat` for this part.
+* The above loop will not reach past the last part unless
+  the number of parts times `htlc_minimum_msat` exceeds the
+  `payment_size_msat`.
+  This case should be checked beforehand and cause the payment
+  to be failed due to too many payment parts.
+
+Once the client has received payment parts that sum up to at
+least `payment_size_msat`, it MUST claim at least one payment part.
+Once that is done, the protocol flow is considered complete.
+
+The parts that the client receives are non-standard, as the amount
+in the onion would not be the same as the amount sent by the LSP.
+Clients MUST suppress such checks for parts it recognizes as having
+been received to pay for a JIT Channel open.
+Clients would need some mechanism to recognize such payments, such as
+by flagging such payments in its invoice database, putting such
+payments in a separate invoice database, or adding special flags inside
+the `metadata` or `payment_secret` of the invoice, or by changing how
+`payment_secret` is computed based on whether the invoice is to pay for
+a JIT Channel or not and trying both ways.
+
+### 6.  Post-opening Normal Operation
+
+After the channel has been opened and an `alias` has been specified,
+the client MUST use the `alias` for future invoices on the
+channel, as normal for users of zero-confirmation channels.
+Note that use of `alias` forces the `alias` to always be used, even
+after the channel is confirmed.
+
+Invoices issued after the channel open MUST use the feerates
+negotiated for the new channel.
+
+> **Rationale** While it is desirable that the client would be able
+> to issue multiple invoices before the channel is opened, this may
+> require that the `alias` be identical to the `jit_channel_scid`,
+> which might not be feasible under some setups.
+> In particular, `jit_channel_scid` may be stored in a database that
+> has strict time limits from the `opening_fee_params.valid_until`,
+> while the `alias` SCID would remain valid while the channel is
+> open.
+> In addition, the "normal" Lightning base and proportional channel
+> fees would need to be applied to the initial payment(s).
+> This version of this API therefore requires that the client first
+> be paid once via the `jit_channel_scid` before it can be issue
+> further invoices with the `alias`.
+>
+> Future versions of this API may allow for more flexibility on the
+> client side, including support for variable-amount invoices,
+> multiple incoming payments being stalled until the `opening_fee`
+> is achieved and the channel can be opened, and so on.
+
+LSPs MUST consider the `jit_channel_scid` as yet another alias
+for the specified client, up to until the `valid_until` time
+selected.
+LSPs MUST consider this alias not just for forwarded payments, but
+also for onion messages.
```

Comment:
Do we have a section about Implementation details that references the specific implementation apis to implement this spec? Similar to the JSON RPC spec?